apps/teams-test-app/src/components/privateApis/ExternalAppCommandsAPIs.tsx

```diff
@@ -0,0 +1,58 @@
+import { externalAppCommands } from '@microsoft/teams-js';
+import React from 'react';
+
+import { ApiWithoutInput, ApiWithTextInput } from '../utils';
+import { ModuleWrapper } from '../utils/ModuleWrapper';
+
+const CheckExternalAppCommandsCapability = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'checkExternalAppCommandsCapability',
+    title: 'Check External App Commands Capability',
+    onClick: async () =>
+      `External App Commands module ${externalAppCommands.isSupported() ? 'is' : 'is not'} supported`,
+  });
+
+const ProcessActionCommand = (): React.ReactElement =>
+  ApiWithTextInput<{
+    appId: string;
+    commandId: string;
+    extractedParameters: Record<string, string>;
+  }>({
+    name: 'processActionCommand',
+    title: 'Process Action Command',
+    onClick: {
+      validateInput: (input) => {
+        if (!input.appId) {
+          throw new Error('appId is required');
+        }
+        if (!input.commandId) {
+          throw new Error('commandId is required');
+        }
+        if (!input.extractedParameters) {
+          throw new Error('extractedParameters is required');
+        }
+      },
+      submit: async (input) => {
+        const result = await externalAppCommands.processActionCommand(
+          input.appId,
+          input.commandId,
+          input.extractedParameters,
+        );
+        return JSON.stringify(result);
+      },
+    },
+    defaultInput: JSON.stringify({
+      appId: 'b7f8c0a0-6c1d-4a9a-9c0a-2c3f1c0a3b0a',
+      commandId: 'command-Id',
```

Comment:
Please update this to commandId field with value `testCommandId` to be more meaningful and get aligned with change I suggested to Lakhveer in web host sdk side.

Reply:
Sure, updated!